frontend/language/src/nb.json

```
@@ -182,6 +192,7 @@
   "app_settings.about_tab_enterprise_user_field_label": "Virksomhetsbruker",
   "app_settings.about_tab_enterprise_user_show_text": "Appen har {{shouldText}} Virksomhetsbruker aktivert.",
   "app_settings.about_tab_error_available_for_type": "Du må velge minst en part",
```

Somewhat out of scope, but what do you think about accentuating the word `en`?
```suggestion
  "app_settings.about_tab_error_available_for_type": "Du må velge minst én part",
```